Mathlib/SetTheory/Ordinal/Basic.lean

```diff
@@ -1364,6 +1364,11 @@ theorem small_iff_lift_mk_lt_univ {α : Type u} :
   · rintro ⟨c, hc⟩
     exact ⟨⟨c.out, lift_mk_eq.{u, _, v + 1}.1 (hc.trans (congr rfl c.mk_out.symm))⟩⟩
 
+/-- If a cardinal `c` is non zero, then `c.ord.toType` has a least element. -/
+noncomputable def orderBotToType (c : Cardinal) (hc : c ≠ 0) :
```

Comment:
This should match `Ordinal.toTypeOrderBotOfPos` in both name and arguments. Though, I wouldn't mind changing the latter into this, since its current name is more unwieldy and it's slightly easier to get `x ≠ 0` than `0 < x` (particularly now that we don't yet have a `CanonicallyOrderedAdd` instance for `Ordinal`).

Reply:
Yeah, I think taking in the != 0 argument is better in both cases. 
To me `orderBotToType` is a slightly better name, it certainly matches the symbol read of the statement. Would you mind adjusting `Ordinal.toTypeOrderBotOfPos` closer to the name here?

Reply:
I have deprecated `Ordinal.toTypeOrderBotOfPos`, and I think the new definition may just be named `Ordinal.toTypeOrderBot`.